### DISCUSSION TITLE
styled-components-breakpoint webpack error in NextJS

### DISCUSSION THREAD

**Discussion by arthurpeck** [vercel/next.js#12249] [Help]
Hi, 
After sucessfully used the styled-components-breakpoint in React in the past, I wanted to add styled-components-breakpoint to my components in NextJS. 
When I use it, it throws an webpack error:

`TypeError: styled_components_breakpoint__WEBPACK_IMPORTED_MODULE_4___default(...) is not a function`

Component:

```
import breakpoint from 'styled-components-breakpoint';

const Logo = styled.img`
    display: block;

    ${breakpoint('mobile')`
        width: 100%;
    `}

    ${breakpoint('tablet')`
        width: 60%;
    `}

    ${breakpoint('desktop')`
        width: 30%;
    `}
`;
```

next.config.js:
```
// next.config.js
const withOptimizedImages = require('next-optimized-images');
const path = require('path');
module.exports = withOptimizedImages({
  webpack(config) {
    config.resolve.alias.images = path.join(__dirname, 'public/images');
    return config;
  },
});

```
Why is it throwing an error? I can use styled-components perfectly without breakpoints and webpack compiles...

**Comment by ebs-PriyankaKanwar** [comment]
we have updated next JS, all things are working fine except one the CSS is not loaded in proper order in production mode, however, it's working fine in dev mode.

Here are the versions
Node: v13.7.0
React:^16.12.0
NextJs:9.1.6
OS: Linux - ubuntu

Here is what I got when did build:

├ static/pages/_app.js 16.1 kB
├ chunks/1899c3d08f1d4a9ab6e4e22050e66cd1600464e8.b9ea19.js 3.53 kB
├ chunks/48183758.ede116.js 65 B
├ chunks/57a5805a715b88beb8aec3ac43d8f962ef8879cc.53c4e4.js 2.34 kB
├ chunks/9ed67fcaced240be487e7a2880bfff868999eacb.76e83a.js 8.29 kB
├ chunks/ab68c4ca07cacf4e3048ee2d6f69810106b4b489.68a8b3.js 10.4 kB
├ chunks/d32b6132cd0996134c9f87f7274f553f791cd563.18dde5.js 6.39 kB
├ chunks/d6e4d54ae3ad4e326943499c7720eb830c61721b.b7693f.js 4.41 kB
├ chunks/framework.4dd100.js 40.3 kB
├ runtime/main.d3f0e8.js 13.2 kB
├ runtime/webpack.361a6a.js 1.59 kB
├ css/16c4518f007189356065.css 54.9 kB
└ css/f93669f8200e9712e425.css 30.6 kB

Here is my next.config :
const withFonts = require('next-fonts');
const withImages = require('next-images');
// const withCSS = require('@zeit/next-css')

const withPlugins = require('next-compose-plugins');
const nextEnv = require('next-env');
const dotenvLoad = require('dotenv-load');

dotenvLoad();

withFonts({
webpack(config, options) {
// config.plugins.push(new webpack.EnvironmentPlugin(localEnv))
return config;
}
})

module.exports = withPlugins([

nextEnv({
staticPrefix: 'CUSTOM_STATIC_',
publicPrefix: 'CUSTOM_PUBLIC_',
}),withFonts, withImages
]);

I have two things here
1 - why we are having two files of CSS when created build? How can we merge into one CSS file?
2 - How I can fix it so that it will work in the same manner as in dev mode?

Any suggestions will be helpful.
